generate_config.sh

```diff
@@ -163,6 +193,17 @@ git checkout -f $git_branch
 
 [ ! -f ./data/conf/rspamd/override.d/worker-controller-password.inc ] && echo '# Placeholder' > ./data/conf/rspamd/override.d/worker-controller-password.inc
 
+if [[ "${CONTAINER_ENGINE}" == "podman" ]]; then
+    MAILCOW_HTTP_BIND="127.0.0.1"
+    MAILCOW_HTTPS_BIND="127.0.0.1"
+
+    # Apply patches for usage with Podman
+    bash ./patches-for-podman.sh
+else
+    MAILCOW_HTTP_BIND=""
+    MAILCOW_HTTPS_BIND=""
+fi
```

Comment:
Is this really needed? This will prevent mailcows UI being reachable over IPv6 by default and any manual configurations.

IPv6 will only be available if the user sets the IPv6 according to this:
https://docs.mailcow.email/post_installation/firststeps-ip_bindings/#ipv6-binding